appserver/common/glassfish-ee-api/src/main/java/com/sun/appserv/ClassLoaderUtil.java

```diff
@@ -300,6 +251,89 @@ public static void releaseLoader(URLClassLoader classLoader) {
     }
     
     /**
+     *Initializes the class.
+     *<p>
+     *Each utility method should invoke init() before doing their own work
```

Comment:
Should be spaces between the * and text - like you did above.

Reply:
Yeah indeed, I did some base formatting but seemingly didn't do all of it